src/helpers/time.js

```diff
@@ -0,0 +1,30 @@
+let moment = require('moment');
```

Comment:
@Chirag-Khatri-CK this is nodeJS import style, it won't work here. You need to use  `import moment from 'moment';`